src/langchain_google_firestore/utility/document_converter.py

```diff
@@ -0,0 +1,179 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+import ast
+
+from typing import (
+    TYPE_CHECKING,
+    List,
+)
+
+from langchain_core.documents import Document
+
+IMPORT_ERROR_MSG = (
+    "`firestore` package not found, please run `pip3 install google-cloud-firestore`"
+)
+
+if TYPE_CHECKING:
+    from google.cloud.firestore_v1.client import Client
+    from google.cloud.firestore_v1.base_document import DocumentSnapshot
+    from google.cloud.firestore_v1.document import DocumentReference
+
+
+class DocumentConverter:
+
+    @staticmethod
+    def convertFirestoreDocument(
```

Comment:
In this case I'm not sure what's the better practice. This method does not require any member variable. It could be outside of the class as some package method.

I followed more of the static utility classes pattern but if you think it all of these should be without any class and just in this file I can change it.